kotlin/workflow-core/src/main/java/com/squareup/workflow/EventHandler.kt

```diff
@@ -0,0 +1,47 @@
+package com.squareup.workflow
+
+/**
+ * TODO kdoc
+ */
+abstract class EventHandler<in EventT> : (EventT) -> Unit {
```

Comment:
Does this really need to be an abstract class? Seems like it could be final and wrap a function pointer. I ask b/c its so dicey to have equality of mismatching types, and seems unnecessary.

Reply:
Good point. Will make final.